### PR TITLE
hpc: Add test for enable HPC through yast

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
@asmorodskyi

**Comment by Soulofdestiny** [comment]
verification: opeth.suse.de/tests/5183#live

File: tests/hpc/enable_in_zypper.pm [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+use warnings;
+use testapi;
+use utils;
+use lockapi;
+
+sub run() {
+
+    select_console 'root-console';
+    # disable packagekitd
+    script_run 'systemctl mask packagekit.service';
```
Comment (asmorodskyi) [inline tests/hpc/enable_in_zypper.pm]: please replace with pkcon_quit from utils

File: tests/hpc/enable_in_zypper.pm [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    script_run 'systemctl stop packagekit.service';
+
+    my $repo     = get_required_var('HPC_REPO');
+    my $reponame = "Devel_HPC_SLE12SP2";
```
Comment (asmorodskyi) [inline tests/hpc/enable_in_zypper.pm]: if we store URL to repo in Test Suite variable probably we can do the same for reponame ?

**Comment by Soulofdestiny** [comment]
updated

**Pull request merged**